source-shopify-native/source_shopify_native/resources.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime, timedelta, UTC
+import functools
+from logging import Logger
+
+from estuary_cdk.flow import CaptureBinding, ValidationError
+from estuary_cdk.capture import common, Task
+from estuary_cdk.http import HTTPMixin, TokenSource, HTTPError
+
+from .models import (
+    EndpointConfig,
+    ResourceConfig,
+    ResourceState,
+    TimestampedResource,
+    OAUTH2_SPEC,
+)
+from .api import (
+    fetch_products,
+)
+
+from .graphql.bulk_job_manager import BulkJobManager
+
+
+AUTHORIZATION_HEADER = "X-Shopify-Access-Token"
+
+
+async def validate_credentials(
+        log: Logger, http: HTTPMixin, config: EndpointConfig
+):
+    """
+    Validates that the provided access token is a valid Front access token.
```

Comment:
Docstring needs updated. TBH I'd just remove it entirely since the function is self-explanatory.